gzip_logs_tools/log_compressor/log_compressor.c

```diff
@@ -564,6 +565,11 @@ init_unix_dgram_socket(state_t* state, const char* path, const char* owner)
 	}
 	
 	unlink(path);
+	if (strlen(path) > sizeof(struct sockaddr_un) - 1)
```

Comment:
can switch back to addr.sun_path, it's better since that's what we use in strncpy. 
also, in general, i usually prefer sizeof on vars over sizeof on types - if you change the var type, it doesn't break.